packages/terra-card/src/Card.scss

```diff
@@ -0,0 +1,18 @@
+:local {
+  /* stylelint-disable selector-class-pattern  */
+  .card-container {
+    background-color: var(--terra-card-background-color, #fff);
+    border-radius: var(--terra-card-border-radius, 3px);
+    box-shadow: var(--tera-card-box-shadow, 0 1px 1px 0 #1c1f21);
+  }
```

Comment:
I think you will want overflow hidden. If something with a square corner butts up against the edge (like a status component on the left) you want to contain it within the curved border radius.